relationships/synthesis/EXT-SERVICE-to-INFRA-ELASTICACHEREDISCLUSTER.yml

```diff
@@ -1,5 +1,29 @@
 relationships:
```

Comment:
Following the `DATABASE` example, this file should be named `EXT-SERVICE-to-REDIS.yml`.